Extract creates bad parquet-file

The error indicates that the process exits before the file is closed?

```
@ingestionTime
1673609394973
@log
671150066425:/aws/lambda/Level1AStack-Level1ALambdaADFD36D4-EAWreaV1f5vm
@logStream
2023/01/13/[$LATEST]0dedabdf41a847fe937b548911b447e5
@message
[ERROR] Level1AException: Failed to initialize handler: Could not open Parquet input source 'ops-payload-level0-v0.1/CCD/2023/1/9/MATS_OPS_Level0_VC1_APID100_20230108-141145_20230109-141925.parquet': Parquet magic bytes not found in footer. Either the file is corrupted or this is not a parquet file.
Traceback (most recent call last):
  File "/var/task/handlers/level1a.py", line 231, in lambda_handler
    raise Level1AException(f"Failed to initialize handler: {err}") from err
@timestamp
1673609393631
```

--- a/raclambda/raclambda/raclambda_stack.py
+++ b/raclambda/raclambda/raclambda_stack.py
@@ -69,8 +69,8 @@ class RacLambdaStack(Stack):
             timeout=lambda_timeout,
             architecture=lambda_.Architecture.X86_64,
             runtime=lambda_.Runtime.PYTHON_3_9,
-            memory_size=1024,
-            ephemeral_storage_size=Size.mebibytes(1024),
+            memory_size=4096,
+            ephemeral_storage_size=Size.mebibytes(2048),
             environment={
                 "RAC_DREGS": dregs_bucket.bucket_name,
                 "RAC_OUTPUT": output_bucket.bucket_name,

